DiceKitTests/MultiplicationExpression_Tests.swift

```diff
@@ -162,13 +162,31 @@ extension MultiplicationExpression_Tests {
 // MARK: - CustomStringConvertible
 extension MultiplicationExpression_Tests {
     
-    func test_CustomStringConvertible() {
+    func test_CustomStringConvertible_constantAndDie() {
         let expression = 2 * d(10)
-        let expected = "2d10"
+        let expected = "\(String(c(2)))\(String(d(10)))"
```

Comment:
For consistency should we always use either `\(String(c(2))` or `\(c(2))`? I'm assuming you did the former to break up the parenthesis a little, but I noticed we used the former for this test, and the later for the result type test. Just an observation if anything.